uaclient/jobs/update_messaging.py

```diff
@@ -390,3 +397,25 @@ def refresh_motd():
             logging.exception(exc)
 
     system.subp(["sudo", "systemctl", "restart", "motd-news.service"])
+
+
+def update_contract_expiry(cfg: config.UAConfig):
+    orig_token = cfg.machine_token
+    machine_token = orig_token["machineToken"]
+    contract_id = orig_token["machineTokenInfo"]["contractInfo"]["id"]
+    contract_client = contract.UAContractClient(cfg)
```

Comment:
Just to be safe, let's add an exception handling here for the case where we don't have those keys in the dict

Reply:
I have updated and used `.get` method to cover any exceptions from `KeyError` exceptions